### ISSUE TITLE
@-webkit-keyframes should be supported 

### ISSUE THREAD

**Issue by nbeloglazov** [ampproject/amphtml#1475]
According to to AMP [specification](https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#properties) both `@keyframes` and `@-vendorPrefix-keyframes` should be supported. But validator complains `saw invalid at rule '-webkit-keyframes'`. 

Test page:
```html
<!doctype html>
<html ⚡>
<head>
  <meta charset="utf-8">
  <link rel="canonical" href="./regular-html-version.html" />
  <meta name="viewport" content="width=device-width,minimum-scale=1">
  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
  <style amp-custom>
    body {
      background: black;
    }
    @-webkit-keyframes a {
        0%, to {
            -webkit-transform: scale(.7);
            transform: scale(.7);
            background-color: rgba(0, 0, 0, .3)
        }
        50% {
            -webkit-transform: scale(.8);
            transform: scale(.8);
            background-color: rgba(0, 0, 0, .5)
        }
    }
    @keyframes a {
        0%, to {
            -webkit-transform: scale(.7);
            transform: scale(.7);
            background-color: rgba(0, 0, 0, .3)
        }
        50% {
            -webkit-transform: scale(.8);
            transform: scale(.8);
            background-color: rgba(0, 0, 0, .5)
        }
    }
  </style>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
</head>
<body>
Hello, world.
</body>
</html>
```

Removing `-webkit-keyframes` fixes the problem.   

Thanks,
Nikita

**Comment by cramforce** [comment]
Working on fixing this already. CC @erwinmombay @Gregable

**Comment by erwinmombay** [comment]
https://github.com/ampproject/amphtml/issues/1315

**Comment by erwinmombay** [comment]
lets close this one and continue on the other thread.

**Comment by nbeloglazov** [comment]
@cramforce, @erwinmombay could either of you update #1315 once prefix-support issue is fixed? As I understand #1315 is about `@support` and not really prefixes. Thanks!

**Comment by cramforce** [comment]
Updated the title of #1315, but we can ping here as well.
